src/main/frontend/util.cljc

```diff
@@ -907,6 +907,13 @@
   []
   #?(:cljs (tc/to-long (cljs-time.core/now))))
 
+; Returns the milliseconds representation of the provided time, in the local timezone.
+; For example, if you run this function at 10pm EDT in the EDT timezone on May 31st,
+; it will return 1622433600000, which is equivalent to Mon May 31 2021 00 :00:00.
+(defn today-at-local-ms
+  [hours mins secs millisecs]
+  (.setHours (js/Date. (.now js/Date)) hours mins secs millisecs))
```

Comment:
The `js/Date` call caused this error, the full log is at https://github.com/logseq/logseq/pull/2061/checks?check_run_id=2715788366
```
failed to require macro-ns "frontend.util", it was required by "frontend.state"
Error in phase :compile-syntax-check
RuntimeException: No such namespace: js
```
We might need to specify the environment for the fn `today-at-local-ms`, something like `#?(:cljs (defn today-at-local-ms ...))`.

Reply:
Ah, thank you. I just pushed a fix for this.

Reply:
Thanks!